metashare/api/serializers.py

```diff
@@ -91,6 +91,7 @@ class Meta:
     def get_branch_url(self, obj) -> Optional[str]:
         if obj.branch_name:
             return f"{obj.repository.repo_url}/tree/{obj.branch_name}"
+        return None
```

Comment:
I was actually discussing this with some friends, and was going to put in this change. I was trying out type annotation to indicate that the `None` branch was expected, but I think that including an explicit `None` is ultimately better.